micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/DefaultPropertySourceLoader.java

```diff
@@ -32,12 +31,12 @@
  *
  * @author Tobias Schäfer
  */
-public class DefaultPropertySourceLoader implements PropertySourceLoader, Ordered {
+public class DefaultPropertySourceLoader implements PropertySourceLoader {
 
     /**
      * Position for the system property source loader in the chain.
      */
-    protected static final int POSITION = Ordered.LOWEST_PRECEDENCE;
```

Comment:
I think this should actually be Integer.MIN_VALUE so that we have lowest priority.  I changed it in a fixup commit. Please check.